polaris-migrator/src/migrate.ts

```diff
@@ -1,7 +1,6 @@
 import fs from 'fs';
 import path from 'path';
 
-// @ts-expect-error TS can't resolve the type of this import
 import * as jscodeshift from 'jscodeshift/src/Runner';
```

Comment:
I guess TS _can_ resolve these types now, so it told me to remove the comment. Which I did, and there's no error 🌮